cpucounters.cpp

```diff
@@ -1680,6 +1697,14 @@ PCM::~PCM()
     if (instance)
     {
         destroyMSR();
+
+        /* clear vectors, triggering destruction of contained objects */
+        energy_status.clear();
```

Comment:
are explicit calls to clear really required? I thought destructors will be called automatically in PCM::~PCM destructors

Reply:
I am afraid you are right.
I will re-submit the pull request.